staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/compile.go

```diff
@@ -167,7 +168,7 @@ type CompilationResult struct {
 	ExpressionAccessor ExpressionAccessor
 }
 
-// CompileCELExpression returns a compiled CEL expression.
+// CompileCELExpression returns a compiled validating policy CEL expression.
```

Comment:
nit: we don't care here if it is a validating policy cel expression or cel expression in other path right? Asking because it might be used in other path later and people might be confused.

Reply:
Good catch, yes, this is shared by all expression types.